core/src/main/java/io/undertow/server/handlers/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -77,22 +78,22 @@
     private boolean initialRun = true;
     private final boolean rotate;
 
-    public DefaultAccessLogReceiver(final Executor logWriteExecutor, final File outputDirectory, final String logBaseName) {
+    public DefaultAccessLogReceiver(final Executor logWriteExecutor, final Path outputDirectory, final String logBaseName) {
```

Comment:
It looks like there are lots of constructors that are no longer backwards compatible? I think we should also have a File version for compatibility